packages/paypal-commerce-integration/src/paypal-commerce/paypal-commerce-button-strategy.ts

```diff
@@ -174,6 +174,8 @@ export default class PayPalCommerceButtonStrategy implements CheckoutButtonStrat
                     orderDetails,
                 );
 
+            billingAddress.phone = orderDetails?.payer?.phone?.phone_number.national_number || '';
```

Comment:
Why don't you update getBillingAddressFromOrderDetails method with this change?

Reply:
fixed